cmd/kubeops/cmd/root.go

```diff
@@ -52,8 +52,6 @@ func setFlags(c *cobra.Command) {
 	c.Flags().StringVar(&serveOpts.PinnipedProxyCACert, "pinniped-proxy-ca-cert", "", "Path to certificate authority to use with requests to pinniped-proxy service")
 	c.Flags().IntVar(&serveOpts.Burst, "burst", 15, "internal burst capacity")
 	c.Flags().Float32Var(&serveOpts.Qps, "qps", 10, "internal QPS rate")
-	c.Flags().StringVar(&serveOpts.NamespaceHeaderName, "namespace-header-name", "", "name of the header field, e.g. namespace-header-name=X-Consumer-Groups")
-	c.Flags().StringVar(&serveOpts.NamespaceHeaderPattern, "namespace-header-pattern", "", "regular expression that matches only single group, e.g. namespace-header-pattern=^namespace:([\\w]+):\\w+$, to match namespace:ns:read")
```

Comment:
In cobra, when a non-recognized flag is passed, it errors. I'd rather deprecate the flags for now instead of deleting them.

See lines below.